components/google_drive/actions/create-file/create-file.js

```diff
@@ -0,0 +1,248 @@
+const googleDrive = require("../../google_drive.app");
+const fs = require("fs");
+const got = require("got");
+
+module.exports = {
+  key: "google_drive-create-file",
+  name: "Create a New File",
+  description: "Create a new file from a URL or /tmp/filepath.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleDrive,
+    drive: {
+      propDefinition: [
+        googleDrive,
+        "watchedDrive",
+      ],
+    },
+    parents: {
```

Comment:
Should probably be singular (`parent`), since you aren't accepting multiple options here.

Reply:
Updated